test/helpers/TestUtils.php

```diff
@@ -30,21 +30,27 @@ public static function addHost($host)
         file_put_contents('/etc/hosts', $hostFile);
     }
 
-    public static function fetchURL($url, $timeout = 3)
+    public static function fetchURL($url, $timeout = 3, $langCookie = null)
```

Comment:
I'm finding these tests really hard to read with `$langCookie` being a string parameter. I had no idea `fetchURL($url, 3, 'en')` sets the cookie language. Can't we pass in an array of cookies in the test ? `array('wovn_selected_lang' => 'en')` that is very clear.

Reply:
Good point.

Reply:
Might also be a good idea to make `$timeout = null` the default, and do a `= 3` inside the method. Just so we could pass in null instead of an arbitrary 3 inside of tests if we don't care.